### PR TITLE
how-to/deploy: update cluster deploy/update document

### PR DESCRIPTION
**Pull request by liubo0127**
@lilin90 @zhangjinpeng1987 PTAL

### CODE REVIEW

File: dev/how-to/deploy/orchestrated/ansible.md [hunk 1]
```diff
@@ -428,11 +428,17 @@ location_labels = ["host"]
...
-        - `raftdb defaultcf block-cache-size(GB)` = MEM * 80% / TiKV 实例数量 * 2.5% (最小 128 MB)
+    1.  多实例情况下，需要修改 `tidb-ansible/conf/tikv.yml` 中 `block-cache-size` 下面的 `capacity` 参数:
+
+        ```
+        storage:
+          block-cache:
+            capacity: "1GB"
+        ```
+
+        > TiKV 实例数量指每个服务器上 TiKV 的进程数量
```
Comment (lilin90) [inline dev/how-to/deploy/orchestrated/ansible.md]: Do you want to add a Note? If not, please remove the `>` character. If yes, please refer to the **注意** format in other documents and follow it. Thanks！

**Comment by yikeke** [comment]
**Important Update**: We have added a markdownlint static check in the CI check to improve the quality of our documentation, so your unclosed PR might fail the CI check now.

Click `details` beside the `ci/circleci: lint — Your tests failed on CircleCI` message, and you can check the details for the failure and fix all the issues in your PR. It is recommended to install "markdownlint" plugin in your VS Code editor, so it will check all potential issues that fail the CI check. 

If you have any problem with troubleshooting, please consult @yikeke for help. Thanks for your cooperation~

**Comment by yikeke** [comment]
I have fixed the markdownlint issues in this PR. Please address Lilian's comments before you merge it, thanks! @liubo0127

File: dev/how-to/deploy/orchestrated/ansible.md (hunk 1 above)
Comment (liubo0127) [reply dev/how-to/deploy/orchestrated/ansible.md]: Done

File: dev/how-to/deploy/orchestrated/ansible.md [hunk 2]
```diff
@@ -428,11 +428,19 @@ location_labels = ["host"]
...
-        - `raftdb defaultcf block-cache-size(GB)` = MEM \* 80% / TiKV 实例数量 \* 2.5% (最小 128 MB)
+    1. 多实例情况下，需要修改 `tidb-ansible/conf/tikv.yml` 中 `block-cache-size` 下面的 `capacity` 参数:
+
+        ```
+        storage:
+          block-cache:
+            capacity: "1GB"
+        ```
+
+        > **注意**
```
Comment (yikeke) [inline dev/how-to/deploy/orchestrated/ansible.md]:
```suggestion
        > **注意：**
```

File: dev/how-to/deploy/orchestrated/ansible.md [hunk 3]
```diff
@@ -428,11 +428,19 @@ location_labels = ["host"]
...
+        >
+        > TiKV 实例数量指每个服务器上 TiKV 的进程数量
```
Comment (yikeke) [inline dev/how-to/deploy/orchestrated/ansible.md]:
```suggestion
        > TiKV 实例数量指每个服务器上 TiKV 的进程数量。
```

File: dev/how-to/deploy/orchestrated/ansible.md [hunk 4]
```diff
@@ -446,10 +454,22 @@ location_labels = ["host"]
             # low-concurrency: 8
         ```
 
-        - 推荐设置：实例数 \* 参数值 = CPU 核数 \* 0.8。
+        > **注意**
```
Comment (yikeke) [inline dev/how-to/deploy/orchestrated/ansible.md]:
```suggestion
        > **注意：**
```

File: dev/how-to/deploy/orchestrated/ansible.md [hunk 5]
```diff
@@ -446,10 +454,22 @@ location_labels = ["host"]
...
 
     3. 如果多个 TiKV 实例部署在同一块物理磁盘上，需要修改 `conf/tikv.yml` 中的 `capacity` 参数:
-        - `capacity` = 磁盘总容量 / TiKV 实例数量，例如 "100GB"
+
+        ```
+        raftstore:
+          capacity: 0
+        ```
+
+        > **注意**
```
Comment (yikeke) [inline dev/how-to/deploy/orchestrated/ansible.md]:
```suggestion
        > **注意：**
```

File: dev/how-to/upgrade/from-previous-version.md [hunk 6]
```diff
@@ -87,19 +89,39 @@ $ git clone -b $tag https://github.com/pingcap/tidb-ansible.git
...
+    readpool:
+      coprocessor:
+        # Notice: if CPU_NUM > 8, default thread pool size for coprocessors
+        # will be set to CPU_NUM * 0.8.
+        # high-concurrency: 8
+        # normal-concurrency: 8
+        # low-concurrency: 8
+    ```
+
+    > **注意**
```
Comment (yikeke) [inline dev/how-to/upgrade/from-previous-version.md]:
```suggestion
    > **注意：**
```

File: dev/how-to/upgrade/from-previous-version.md [hunk 7]
```diff
@@ -87,19 +89,39 @@ $ git clone -b $tag https://github.com/pingcap/tidb-ansible.git
...
+    >
+    > 单机多 TiKV 实例（进程）情况下，需要修改这三个参数
```
Comment (yikeke) [inline dev/how-to/upgrade/from-previous-version.md]:
```suggestion
    > 单机多 TiKV 实例（进程）情况下，需要修改这三个参数。
```

File: dev/how-to/upgrade/from-previous-version.md [hunk 8]
```diff
@@ -87,19 +89,39 @@ $ git clone -b $tag https://github.com/pingcap/tidb-ansible.git
...
+- TiKV 配置中不同 CF 中的 `block-cache-size` 参数变更为 `block-cache`
+
+    ```
+    storage:
+      block-cache:
+        capacity: "1GB"
+    ```
 
-单机多 TiKV 实例情况下，需要修改这三个参数，推荐设置：`实例数 * 参数值 = CPU 核数 * 0.8`。
+    > **注意**
```
Comment (yikeke) [inline dev/how-to/upgrade/from-previous-version.md]:
```suggestion
    > **注意：**
```

File: dev/how-to/upgrade/from-previous-version.md [hunk 9]
```diff
@@ -87,19 +89,39 @@ $ git clone -b $tag https://github.com/pingcap/tidb-ansible.git
...
+    >
+    > 单机多 TiKV 实例（进程）情况下，需要修改 `capacity` 参数
```
Comment (yikeke) [inline dev/how-to/upgrade/from-previous-version.md]:
```suggestion
    > 单机多 TiKV 实例（进程）情况下，需要修改 `capacity` 参数。
```

File: v2.1/how-to/deploy/orchestrated/ansible.md [hunk 10]
```diff
@@ -450,7 +450,17 @@ location_labels = ["host"]
...
 
     3. 如果多个 TiKV 实例部署在同一块物理磁盘上，需要修改 `conf/tikv.yml` 中的 `capacity` 参数:
-        - `capacity` = 磁盘总容量 / TiKV 实例数量，例如 "100GB"
+
+        ```
+        raftstore:
+          capacity: 0
+        ```
+
+        > **注意**
```
Comment (yikeke) [inline v2.1/how-to/deploy/orchestrated/ansible.md]:
```suggestion
        > **注意：**
```

File: v2.1/how-to/upgrade/from-previous-version.md [hunk 11]
```diff
@@ -21,7 +21,9 @@ category: how-to
 
 ## 在中控机器上安装 Ansible 及其依赖
 
-TiDB-Ansible release-3.0 版本依赖 2.5.14 Ansible 版本（`ansible=2.5.14`），另依赖 Python 模块：`jinja2>=2.9.6` 和 `jmespath>=0.9.0`。为方便管理依赖，新版本使用 `pip` 安装 Ansible 及其依赖，可参照[在中控机器上安装 Ansible 及其依赖](/how-to/deploy/orchestrated/ansible.md#在中控机器上安装-ansible-及其依赖) 进行安装。离线环境参照[在中控机器上离线安装 Ansible 及其依赖](/how-to/deploy/orchestrated/offline-ansible.md#在中控机器上离线安装-ansible-及其依赖)。
+> 如果已经安装了 Ansible 及其依赖，可跳过该步骤。
```
Comment (yikeke) [inline v2.1/how-to/upgrade/from-previous-version.md]:
```suggestion
> **注意：**
>
> 如果已经安装了 Ansible 及其依赖，可跳过该步骤。
```

File: v2.1/how-to/upgrade/from-previous-version.md [hunk 12]
```diff
@@ -87,19 +89,39 @@ $ git clone -b $tag https://github.com/pingcap/tidb-ansible.git
 
 如之前自定义过 TiDB 集群组件配置文件，请参照备份文件修改 `/home/tidb/tidb-ansible/conf` 下对应配置文件。
 
-TiKV 配置中 `end-point-concurrency` 变更为 `high-concurrency`、`normal-concurrency` 和 `low-concurrency` 三个参数：
+**注意以下参数变更**
```
Comment (yikeke) [inline v2.1/how-to/upgrade/from-previous-version.md]:
```suggestion
**注意以下参数变更：**
```

File: v2.1/how-to/upgrade/from-previous-version.md [hunk 13]
```diff
@@ -87,19 +89,39 @@ $ git clone -b $tag https://github.com/pingcap/tidb-ansible.git
...
+    readpool:
+      coprocessor:
+        # Notice: if CPU_NUM > 8, default thread pool size for coprocessors
+        # will be set to CPU_NUM * 0.8.
+        # high-concurrency: 8
+        # normal-concurrency: 8
+        # low-concurrency: 8
+    ```
+
+    > **注意**
```
Comment (yikeke) [inline v2.1/how-to/upgrade/from-previous-version.md]:
```suggestion
    > **注意：**
```

File: v2.1/how-to/upgrade/from-previous-version.md [hunk 14]
```diff
@@ -87,19 +89,39 @@ $ git clone -b $tag https://github.com/pingcap/tidb-ansible.git
...
+    >
+    > 单机多 TiKV 实例（进程）情况下，需要修改这三个参数
```
Comment (yikeke) [inline v2.1/how-to/upgrade/from-previous-version.md]:
```suggestion
    > 单机多 TiKV 实例（进程）情况下，需要修改这三个参数。
```

File: v2.1/how-to/upgrade/from-previous-version.md [hunk 15]
```diff
@@ -87,19 +89,39 @@ $ git clone -b $tag https://github.com/pingcap/tidb-ansible.git
...
+- TiKV 配置中不同 CF 中的 `block-cache-size` 参数变更为 `block-cache`
+
+    ```
+    storage:
+      block-cache:
+        capacity: "1GB"
+    ```
 
-单机多 TiKV 实例情况下，需要修改这三个参数，推荐设置：`实例数 * 参数值 = CPU 核数 * 0.8`。
+    > **注意**
```
Comment (yikeke) [inline v2.1/how-to/upgrade/from-previous-version.md]:
```suggestion
    > **注意：**
```

File: v2.1/how-to/upgrade/from-previous-version.md [hunk 16]
```diff
@@ -87,19 +89,39 @@ $ git clone -b $tag https://github.com/pingcap/tidb-ansible.git
...
+    >
+    > 单机多 TiKV 实例（进程）情况下，需要修改 `capacity` 参数
```
Comment (yikeke) [inline v2.1/how-to/upgrade/from-previous-version.md]:
```suggestion
    > 单机多 TiKV 实例（进程）情况下，需要修改 `capacity` 参数。
```

File: v3.0/how-to/deploy/orchestrated/ansible.md [hunk 17]
```diff
@@ -429,11 +428,19 @@ location_labels = ["host"]
 
 - 服务配置文件参数调整
 
-    1. 多实例情况下，需要修改 `tidb-ansible/conf/tikv.yml` 中的 `block-cache-size` 参数:
-        - `rocksdb defaultcf block-cache-size(GB)` = MEM \* 80% / TiKV 实例数量 \* 30%
-        - `rocksdb writecf block-cache-size(GB)` = MEM \* 80% / TiKV 实例数量 \* 45%
-        - `rocksdb lockcf block-cache-size(GB)` = MEM \* 80% / TiKV 实例数量 \* 2.5% (最小 128 MB)
-        - `raftdb defaultcf block-cache-size(GB)` = MEM \* 80% / TiKV 实例数量 \* 2.5% (最小 128 MB)
+    1. 多实例情况下，需要修改 `tidb-ansible/conf/tikv.yml` 中 `block-cache-size` 下面的 `capacity` 参数:
```
Comment (yikeke) [inline v3.0/how-to/deploy/orchestrated/ansible.md]:
```suggestion
    1. 多实例情况下，需要修改 `tidb-ansible/conf/tikv.yml` 中 `block-cache-size` 下面的 `capacity` 参数：
```

File: v3.0/how-to/deploy/orchestrated/ansible.md [hunk 18]
```diff
@@ -429,11 +428,19 @@ location_labels = ["host"]
...
+
+        ```
+        storage:
+          block-cache:
+            capacity: "1GB"
+        ```
+
+        > **注意**
```
Comment (yikeke) [inline v3.0/how-to/deploy/orchestrated/ansible.md]:
```suggestion
        > **注意：**
```

File: v3.0/how-to/deploy/orchestrated/ansible.md [hunk 19]
```diff
@@ -429,11 +428,19 @@ location_labels = ["host"]
...
+        >
+        > TiKV 实例数量指每个服务器上 TiKV 的进程数量
```
Comment (yikeke) [inline v3.0/how-to/deploy/orchestrated/ansible.md]:
```suggestion
        > TiKV 实例数量指每个服务器上 TiKV 的进程数量。
```

File: v3.0/how-to/deploy/orchestrated/ansible.md [hunk 20]
```diff
@@ -447,10 +454,22 @@ location_labels = ["host"]
             # low-concurrency: 8
         ```
 
-        - 推荐设置：实例数 \* 参数值 = CPU 核数 * 0.8。
+        > **注意**
```
Comment (yikeke) [inline v3.0/how-to/deploy/orchestrated/ansible.md]:
```suggestion
        > **注意：**
```

File: v3.0/how-to/deploy/orchestrated/ansible.md [hunk 21]
```diff
@@ -447,10 +454,22 @@ location_labels = ["host"]
...
 
     3. 如果多个 TiKV 实例部署在同一块物理磁盘上，需要修改 `conf/tikv.yml` 中的 `capacity` 参数:
-        - `capacity` = 磁盘总容量 / TiKV 实例数量，例如 "100GB"
+
+        ```
+        raftstore:
+          capacity: 0
+        ```
+
+        > **注意**
```
Comment (yikeke) [inline v3.0/how-to/deploy/orchestrated/ansible.md]:
```suggestion
        > **注意：**
```

File: v3.0/how-to/upgrade/from-previous-version.md [hunk 22]
```diff
@@ -22,13 +22,15 @@ aliases: ['/docs-cn/op-guide/tidb-v3.0-upgrade-guide/']
 
 ## 在中控机器上安装 Ansible 及其依赖
 
-TiDB-Ansible release-3.0 版本依赖 2.5.14 Ansible 版本（`ansible=2.5.14`），另依赖 Python 模块：`jinja2>=2.9.6` 和 `jmespath>=0.9.0`。为方便管理依赖，新版本使用 `pip` 安装 Ansible 及其依赖，可参照[在中控机器上安装 Ansible 及其依赖](/how-to/deploy/orchestrated/ansible.md#在中控机器上安装-ansible-及其依赖) 进行安装。离线环境参照[在中控机器上离线安装 Ansible 及其依赖](/how-to/deploy/orchestrated/offline-ansible.md#在中控机器上离线安装-ansible-及其依赖)。
+> 如果已经安装了 Ansible 及其依赖，可跳过该步骤。
```
Comment (yikeke) [inline v3.0/how-to/upgrade/from-previous-version.md]:
```suggestion
> **注意：**
>
> 如果已经安装了 Ansible 及其依赖，可跳过该步骤。
```

File: v3.0/how-to/upgrade/from-previous-version.md [hunk 23]
```diff
@@ -88,19 +90,39 @@ $ git clone -b $tag https://github.com/pingcap/tidb-ansible.git
 
 如之前自定义过 TiDB 集群组件配置文件，请参照备份文件修改 `/home/tidb/tidb-ansible/conf` 下对应配置文件。
 
-TiKV 配置中 `end-point-concurrency` 变更为 `high-concurrency`、`normal-concurrency` 和 `low-concurrency` 三个参数：
+**注意以下参数变更**
```
Comment (yikeke) [inline v3.0/how-to/upgrade/from-previous-version.md]:
```suggestion
**注意以下参数变更：**
```

File: v3.0/how-to/upgrade/from-previous-version.md [hunk 24]
```diff
@@ -88,19 +90,39 @@ $ git clone -b $tag https://github.com/pingcap/tidb-ansible.git
...
+    readpool:
+      coprocessor:
+        # Notice: if CPU_NUM > 8, default thread pool size for coprocessors
+        # will be set to CPU_NUM * 0.8.
+        # high-concurrency: 8
+        # normal-concurrency: 8
+        # low-concurrency: 8
+    ```
+
+    > **注意**
```
Comment (yikeke) [inline v3.0/how-to/upgrade/from-previous-version.md]:
```suggestion
    > **注意：**
```

File: v3.0/how-to/upgrade/from-previous-version.md [hunk 25]
```diff
@@ -88,19 +90,39 @@ $ git clone -b $tag https://github.com/pingcap/tidb-ansible.git
...
+    >
+    > 单机多 TiKV 实例（进程）情况下，需要修改这三个参数
```
Comment (yikeke) [inline v3.0/how-to/upgrade/from-previous-version.md]:
```suggestion
    > 单机多 TiKV 实例（进程）情况下，需要修改这三个参数。
```

File: v3.0/how-to/upgrade/from-previous-version.md [hunk 26]
```diff
@@ -88,19 +90,39 @@ $ git clone -b $tag https://github.com/pingcap/tidb-ansible.git
...
+- TiKV 配置中不同 CF 中的 `block-cache-size` 参数变更为 `block-cache`
+
+    ```
+    storage:
+      block-cache:
+        capacity: "1GB"
+    ```
 
-单机多 TiKV 实例情况下，需要修改这三个参数，推荐设置：`实例数 * 参数值 = CPU 核数 * 0.8`。
+    > **注意**
```
Comment (yikeke) [inline v3.0/how-to/upgrade/from-previous-version.md]:
```suggestion
    > **注意：**
```

File: v3.0/how-to/upgrade/from-previous-version.md [hunk 27]
```diff
@@ -88,19 +90,39 @@ $ git clone -b $tag https://github.com/pingcap/tidb-ansible.git
...
+    >
+    > 单机多 TiKV 实例（进程）情况下，需要修改 `capacity` 参数
```
Comment (yikeke) [inline v3.0/how-to/upgrade/from-previous-version.md]:
```suggestion
    > 单机多 TiKV 实例（进程）情况下，需要修改 `capacity` 参数。
```

File: dev/how-to/upgrade/from-previous-version.md [hunk 28]
```diff
@@ -21,13 +21,15 @@ category: how-to
 
 ## 在中控机器上安装 Ansible 及其依赖
 
-TiDB-Ansible release-3.0 版本依赖 2.5.14 Ansible 版本（`ansible=2.5.14`），另依赖 Python 模块：`jinja2>=2.9.6` 和 `jmespath>=0.9.0`。为方便管理依赖，新版本使用 `pip` 安装 Ansible 及其依赖，可参照[在中控机器上安装 Ansible 及其依赖](/how-to/deploy/orchestrated/ansible.md#在中控机器上安装-ansible-及其依赖) 进行安装。离线环境参照[在中控机器上离线安装 Ansible 及其依赖](/how-to/deploy/orchestrated/offline-ansible.md#在中控机器上离线安装-ansible-及其依赖)。
+> 如果已经安装了 Ansible 及其依赖，可跳过该步骤。
```
Comment (yikeke) [inline dev/how-to/upgrade/from-previous-version.md]:
```suggestion
> **注意：**
>
> 如果已经安装了 Ansible 及其依赖，可跳过该步骤。
```

File: dev/how-to/upgrade/from-previous-version.md [hunk 29]
```diff
@@ -87,19 +91,39 @@ $ git clone -b $tag https://github.com/pingcap/tidb-ansible.git
 
 如之前自定义过 TiDB 集群组件配置文件，请参照备份文件修改 `/home/tidb/tidb-ansible/conf` 下对应配置文件。
 
-TiKV 配置中 `end-point-concurrency` 变更为 `high-concurrency`、`normal-concurrency` 和 `low-concurrency` 三个参数：
+**注意以下参数变更**
```
Comment (liubo0127) [inline dev/how-to/upgrade/from-previous-version.md]:
```suggestion
**注意以下参数变更：**
```

**Pull request merged**